projects/packages/forms/src/blocks/contact-form/components/jetpack-field-width.js

```diff
@@ -11,21 +15,25 @@ export default function JetpackFieldWidth( { setAttributes, width } ) {
 			className="jetpack-field-label__width"
 			__nextHasNoMarginBottom={ true }
 		>
-			<BaseControl.VisualLabel>{ __( 'Field Width', 'jetpack-forms' ) }</BaseControl.VisualLabel>
-			<ButtonGroup aria-label={ __( 'Field Width', 'jetpack-forms' ) }>
+			<ToggleGroupControl
+				__next40pxDefaultSize
+				__nextHasNoMarginBottom
+				aria-label={ __( 'Field Width', 'jetpack-forms' ) }
+				isBlock
+				label={ __( 'Field Width', 'jetpack-forms' ) }
+				onChange={ value => setAttributes( { width: value } ) }
+				value={ width }
+			>
 				{ [ 25, 50, 75, 100 ].map( widthValue => {
```

Comment:
It might be good to define this as a constant. So we don't have to create this array every time we render this control.